include/oneapi/math/detail/backends_table.hpp

```diff
@@ -138,78 +138,78 @@ static std::map<domain, std::map<device, std::vector<const char*>>> libraries =
     { domain::lapack,
       { { device::x86cpu,
           {
-#ifdef ONEMKL_ENABLE_MKLCPU_BACKEND
+#ifdef ONEMATH_ENABLE_MKLCPU_BACKEND
               LIB_NAME("lapack_mklcpu")
 #endif
           } },
         { device::intelgpu,
           {
-#ifdef ONEMKL_ENABLE_MKLGPU_BACKEND
+#ifdef ONEMATH_ENABLE_MKLGPU_BACKEND
               LIB_NAME("lapack_mklgpu")
 #endif
           } },
         { device::amdgpu,
           {
-#ifdef ONEMKL_ENABLE_ROCSOLVER_BACKEND
+#ifdef ONEMATH_ENABLE_ROCSOLVER_BACKEND
               LIB_NAME("lapack_rocsolver")
 #endif
           } },
         { device::nvidiagpu,
           {
-#ifdef ONEMKL_ENABLE_CUSOLVER_BACKEND
+#ifdef ONEMATH_ENABLE_CUSOLVER_BACKEND
               LIB_NAME("lapack_cusolver")
 #endif
           } } } },
 
     { domain::rng,
       { { device::x86cpu,
           {
-#ifdef ONEMKL_ENABLE_MKLCPU_BACKEND
+#ifdef ONEMATH_ENABLE_MKLCPU_BACKEND
               LIB_NAME("rng_mklcpu")
 #endif
           } },
         { device::intelgpu,
           {
-#ifdef ONEMKL_ENABLE_MKLGPU_BACKEND
+#ifdef ONEMATH_ENABLE_MKLGPU_BACKEND
               LIB_NAME("rng_mklgpu")
 #endif
           } },
         { device::amdgpu,
           {
-#ifdef ONEMKL_ENABLE_ROCRAND_BACKEND
+#ifdef ONEMATH_ENABLE_ROCRAND_BACKEND
               LIB_NAME("rng_rocrand")
 #endif
           } },
         { device::nvidiagpu,
           {
-#ifdef ONEMKL_ENABLE_CURAND_BACKEND
+#ifdef ONEMATH_ENABLE_CURAND_BACKEND
               LIB_NAME("rng_curand")
 #endif
           } } } },
 
     { domain::sparse_blas,
       { { device::x86cpu,
           {
-#ifdef ONEMKL_ENABLE_MKLCPU_BACKEND
+#ifdef ONEMATH_ENABLE_MKLCPU_BACKEND
               LIB_NAME("sparse_blas_mklcpu")
 #endif
           } },
         { device::intelgpu,
           {
-#ifdef ONEMKL_ENABLE_MKLGPU_BACKEND
+#ifdef ONEMATH_ENABLE_MKLGPU_BACKEND
               LIB_NAME("sparse_blas_mklgpu")
 #endif
           } } } },
 };
 
-static std::map<domain, const char*> table_names = { { domain::blas, "mkl_blas_table" },
-                                                     { domain::lapack, "mkl_lapack_table" },
-                                                     { domain::dft, "mkl_dft_table" },
-                                                     { domain::rng, "mkl_rng_table" },
+static std::map<domain, const char*> table_names = { { domain::blas, "onemath_blas_table" },
```

Comment:
These occurrences really should have been renamed to `onemkl` earlier as they mean to refer to this project. Similarly I much prefer to use `onemath` instead of `math` when possible to clarify we refer to this project and not "mathematics". These variables aren't used often so 3 more characters should be fine. I'll consider changing it if more people ask for it.